rise-financial-es6.js

```diff
@@ -308,14 +311,32 @@
 
       this._goPending = false;
 
-      this._getData(
-        {
-          type: this.type,
-          duration: this.duration,
-        },
-        this._instruments,
-        this.instrumentFields
-      );
+      if ( this._initialGo ) {
```

Comment:
@stulees To avoid this level of nesting, what I've seen people do and what I really like is to add a `return` as the last line in the `if` statement. That way you can avoid the `else` altogether and all the indentation that comes with it. It makes the code more readable.